doc/source/development/contributing.rst

```diff
@@ -194,31 +194,18 @@ Doing 'git status' again should give something like::
     #       modified:   /relative/path/to/file-you-added.py
     #
 
-Finally, commit your changes to your local repository with an explanatory message. pandas
-uses a convention for commit message prefixes and layout.  Here are
-some common prefixes along with general guidelines for when to use them:
-
-* ENH: Enhancement, new functionality
-* BUG: Bug fix
-* DOC: Additions/updates to documentation
-* TST: Additions/updates to tests
-* BLD: Updates to the build process/scripts
-* PERF: Performance improvement
-* TYP: Type annotations
-* CLN: Code cleanup
-
-The following defines how a commit message should be structured.  Please reference the
-relevant GitHub issues in your commit message using GH1234 or #1234.  Either style
-is fine, but the former is generally preferred:
+Finally, commit your changes to your local repository with an explanatory commit
+message of ``< 80`` chars::
```

Comment:
```suggestion
message::
```
I know this already exists now, but since we squash commits, not very important, and we can simplify things a bit.